src/auth/jwt.strategy.ts

```diff
@@ -3,12 +3,21 @@ import { PassportStrategy } from '@nestjs/passport';
 import { Injectable } from '@nestjs/common';
 import { ConfigService } from '@nestjs/config';
 import fs from 'fs';
+import { Request } from 'express';
 
 @Injectable()
 export class JwtStrategy extends PassportStrategy(Strategy) {
   constructor(configService: ConfigService) {
     super({
-      jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
+      jwtFromRequest: ExtractJwt.fromExtractors([
+        ExtractJwt.fromAuthHeaderAsBearerToken(),
+        (req: Request) => {
```

Comment:
Nice, I guess the multipleExtractors means that this is backwards compatible with `iam-client-lib` versions that were sending the jwt as a bearerToken?

Reply:
Yes it is, that was the main goal :)